playbot/playbot.py

```diff
@@ -86,14 +84,15 @@ def get_data(self):
                     self.heartbeat = 0
                     return buffor.split('\x04')[0]
         except BlockingIOError:
-            self.game = True
+            return buffor
         except ConnectionResetError as e:
-            self.log(e.strerror)
-            self.game = False
+            buffor = e.strerror
         except ConnectionAbortedError as e:
-            self.log(e.strerror)
-            self.game = False
-        return buffor
+            buffor = e.strerror
+
+        self.log(buffor)
```

Comment:
Please don't use buffor variable to store error message in it.
That's not her purpouse.
Also I'm not a fan of more than one return in method (you have three here) but I can live with that.

If you have identical except block for two or more exceptions then you can combine it to one :)
```
except (ConnectionResetError, ConnectionAbortedError) e:
    self.log(e.strerror)
```
https://docs.python.org/3/tutorial/errors.html#handling-exceptions

Reply:
fixed.